plugins/python-build/bin/python-build

```diff
@@ -2118,11 +2118,28 @@ if [ -z "${EZ_SETUP_URL}" ]; then
 fi
 if [ -z "${GET_PIP_URL}" ]; then
   if [ -n "${PIP_VERSION}" ]; then
+    { colorize 1 "WARNING"
+      echo ": pip is going to abandon the support for the custom version installation via custom get-pip URL. The use of PIP_VERSION=${PIP_VERSION} may not work anymore."
```

Comment:
Honestly speaking, I don't have correct words for explanation of this warning.... 😥  Could someone have any idea on this?

Reply:
How does this sound?

```sh
Setting PIP_VERSION=${PIP_VERSION} is no longer supported and may cause failures during the install process.
```